pilot/docker/envoy_istio_telemetry_auth.json

```diff
@@ -0,0 +1,174 @@
+                     "cluster": "mixer_server",
+                     "timeout_ms": 0,
+                     "opaque_config": {
+                       "destination.service": "istio-telemetry.${POD_NAMESPACE}.svc.cluster.local",
+                       "mixer_check": "off",
+                       "mixer_forward": "off",
+                       "mixer_report": "on"
+                     },
+                     "decorator": {
+                       "operation": "Report"
+                     }
+                   }
+                 ]
+               }
+             ]
+           },
+           "filters": [
+             {
+               "type": "decoder",
+               "name": "mixer",
+               "config": {
+                 "v2": {
+                   "defaultDestinationService": "istio-telemetry.${POD_NAMESPACE}.svc.cluster.local",
+                   "mixerAttributes": {
+                     "attributes": {
+                       "destination.uid": {
+                         "stringValue" : "kubernetes://${POD_NAME}.${POD_NAMESPACE}"
+                       }
+                     }
+                   },
+                   "serviceConfigs": {
+                     "istio-telemetry.${POD_NAMESPACE}.svc.cluster.local": {
+                       "disableCheckCalls": true,
+                       "disableReportCalls": false,
+                       "mixerAttributes": {
+                         "attributes": {
+                           "destination.service": {
+                             "stringValue": "istio-telemetry.${POD_NAMESPACE}.svc.cluster.local"
+                           },
+                           "destination.uid": {
+                             "stringValue" : "kubernetes://${POD_NAME}.${POD_NAMESPACE}"
+                           }
+                         }
+                       }
+                     }
+                   },
+                   "transport": {
+                     "reportCluster": "mixer_server"
+                   }
+                 }
+               }
+             },
+             {
+               "type": "",
+               "name": "cors",
+               "config": {}
+             },
+             {
+               "type": "decoder",
+               "name": "router",
+               "config": {}
+             }
+           ],
+           "access_log": [
+             {
+               "path": "/etc/istio/proxy/envoy_access_log.txt"
+             }
+           ]
+         }
+       }
+     ],
+     "ssl_context": {
+       "cert_chain_file": "/etc/certs/cert-chain.pem",
+       "private_key_file": "/etc/certs/key.pem",
+       "ca_cert_file": "/etc/certs/root-cert.pem",
+       "require_client_certificate": true
+     },
+     "bind_to_port": true
+   }
+  ],
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "address": "tcp://127.0.0.1:15000"
+  },
+  "cluster_manager": {
+    "clusters": [
+      {
+        "name": "in.9091",
```

Comment:
I don't think you need two clusters here... You could use the name of one cluster in both places.

Reply:
Done. I just needed to add `"features": "http2"` here, but otherwise adopted this suggestion. Thanks.

Reply:
Thanks. I don't think I can come up with a reason to have two separate upstreams, so this is the right thing to do.